src/plots/cartesian/graph_interact.js

```diff
@@ -2099,8 +2099,7 @@ fx.dragElement = function(options) {
         if(options.doneFn) options.doneFn(gd._dragged, numClicks);
 
         if(!gd._dragged) {
-            var e2 = document.createEvent('MouseEvents');
-            e2.initEvent('click', true, true);
+            var e2 = new MouseEvent('click', { 'button': e.button });
```

Comment:
Sorry guys, I forgot you have to support crappy IE9, next time I'll make sure to comply with that as well. Anyways, if it works... :)